corehq/apps/dashboard/templates/dashboard/dashboard_domain.html

```diff
@@ -55,7 +57,14 @@
         {% endfor %}
     </div>
     <p class="lead text-center" style="margin-bottom: 5em;">
-        {% if project.commtrack_enabled %}
+        {% if is_icds %}
+            {% blocktrans %}
+                Thanks for using ICDS-CAS! If you have questions, suggestions,
+                or other thoughts, <br />we invite you to join the discussion on the
+                <a href="https://groups.google.com/forum/#!forum/commcare-users/join"
+                   target="_blank">CommCare Users Mailing List</a>.
```

Comment:
I suspect they might not want the content related to commcare users. @sravfeyn @mkangia do you know? seems easier to just omit.